installer/stage/BeaKer/install_beaker.sh

```diff
@@ -56,7 +56,10 @@ install_docker () {
 ensure_env_file_exists () {
     $SUDO mkdir -p "$BEAKER_CONFIG_DIR"
 
-    if [ ! -f "$BEAKER_CONFIG_DIR/env" ]; then
+    if [ "$acm_no_interactive" != 'yes' ] && [ ! -f "$BEAKER_CONFIG_DIR/env" ]; then
```

Comment:
This function failed when installing Beaker initially without passing `--no-interactive` to `install_acm.sh`

I believe you want to use `=` rather than `!=` in the first condition.

Reply:
Fixed in https://github.com/activecm/BeaKer/pull/63/commits/e86339305c6b775bb3b440c6c94e65f6e6254486